src/firmware/boards/legacy_robot_stm32f4/io/receive.c

```diff
@@ -367,28 +348,28 @@ void handle_camera_packet(uint8_t *dma_buffer, uint8_t buffer_position)
 
 void handle_other_packet(uint8_t *dma_buffer, size_t frame_length)
 {
-    // printf("got a message with purpose: %i", dma_buffer[MESSAGE_PURPOSE_ADDR]);
-    // printf("var index: %i", dma_buffer[MESSAGE_PURPOSE_ADDR + 1]);
-    // printf("value: %i", dma_buffer[MESSAGE_PURPOSE_ADDR + 2]);
-    switch (dma_buffer[MESSAGE_PURPOSE_ADDR])
+    // printf("got a message with purpose: %i", dma_buffer[MESSAGE_PURPOSE_INDEX]);
+    // printf("var index: %i", dma_buffer[MESSAGE_PURPOSE_INDEX + 1]);
+    // printf("value: %i", dma_buffer[MESSAGE_PURPOSE_INDEX + 2]);
+    switch (dma_buffer[MESSAGE_PURPOSE_INDEX])
     {
         case 0x00:
             if (frame_length == HEADER_LENGTH + 4U + FOOTER_LENGTH)
             {
-                uint8_t which  = dma_buffer[MESSAGE_PAYLOAD_ADDR];
-                uint16_t width = dma_buffer[MESSAGE_PAYLOAD_ADDR + 2U];
+                uint8_t which  = dma_buffer[MESSAGE_PAYLOAD_INDEX];
+                uint16_t width = dma_buffer[MESSAGE_PAYLOAD_INDEX + 2U];
                 width <<= 8U;
-                width |= dma_buffer[MESSAGE_PAYLOAD_ADDR + 1U];
+                width |= dma_buffer[MESSAGE_PAYLOAD_INDEX + 1U];
```

Comment:
Kind of a nit, but can we get defines for these message offsets?

Reply:
I'm going to say that's out of scope here because:
- I didn't change these lines, just the variable that's used in them
- Frankly I don't even know what these offsets are. They're custom encodings for custom test messages :crying_cat_face: